packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md

```diff
@@ -131,6 +131,15 @@ a ?? (b && c && d);
 
 **_NOTE:_** Errors for this specific case will be presented as suggestions (see below), instead of fixes. This is because it is not always safe to automatically convert `||` to `??` within a mixed logical expression, as we cannot tell the intended precedence of the operator. Note that by design, `??` requires parentheses when used with `&&` or `||` in the same expression.
 
+### `ignorePrimitives`
```

Comment:
[Docs] Could you add some examples please? We generally try to -at least for newly added rule options- so users can see exactly how the code looks.

Reply:
I tried to look at the style for examples with options, this rule seems unique in how the examples are presented but decided to keep it the same since I've noticed bradzacher did the same

Reply:
Heh yeah we'll eventually clean all these up... eventually. Thanks!